Include/Python.h

```diff
@@ -146,4 +146,8 @@
 #include "cpython/pyfpe.h"
 #include "tracemalloc.h"
 
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030B0000
+PyAPI_DATA(const long int) Py_Version;
```

Comment:
In the Python code base, we just write "long". Please use "long" here for consistency.

```suggestion
PyAPI_DATA(const long) Py_Version;
```